### ISSUE TITLE
Does the effect persist if launcher quits?

### ISSUE THREAD

**Issue by Barafu** [FeralInteractive/gamemode#160]
Sorry to bother you with a question, but no one else seems to know for sure.

If I run `gamemoderun wine launcher.exe`, and launcher starts a separate .exe and quits immediately, does gamemode effect persists on the game?

Thanks!

P.S. Is there a way to check that gamemode is active on a specific process so that I do not need to bother with such questions again?

**Comment by Barafu** [comment]
To clarify: I know that if I enable bell in configs, it does not notify that "gamemode ended" untill the game itself quits. But this does not really mean for sure that all tweaks are applied, does it?

**Comment by gicmo** [comment]
I think it should probably work because gamemode will get turned on for the wine process (gamemode run uses `libgamemodeauto.so`, which gets preloaded to the wine processes and all host children-process because `LD_PRELOAD` should set for all of them).

The quickest way is to check the journal `journalctl --user -r -u gamemoded`. 

I am also working on a small tool ([gamemode-explorer](https://github.com/gicmo/gamemode-explorer) that monitors what is happening, but it needs gamemode master and is not super verbose/helpful yet).

**Comment by aejsmith** [comment]
Yes, I think it should as well. You can also use `systemctl --user status gamemoded` - the "Status" line will tell you whether or not it's currently active.

**Comment by Sanaki** [comment]
Interestingly, `systemctl --user status gamemoded` shows inactive while `gamemoded -s` shows active, when loading via prepending a lutris game with gamemoderun.

**Comment by mdiluz** [comment]
> Interestingly, systemctl --user status gamemoded shows inactive while gamemoded -s shows active, when loading via prepending a lutris game with gamemoderun.
> 
> EDIT: After fixing my nvidia prime setup to actually work properly with vsync, both commands show active.

@Sanaki This is very odd, as NVIDIA prime shouldn't have anything to do with gamemode. If you see anything like this again then please let us know!

**Comment by Sanaki** [comment]
Part of that chain of nuttery was compiling the 32-bit libraries for gamemode, that may be more relevant in this case.
